### PR TITLE
Add instructions on how to set bash as your default shell

### PR DESCRIPTION
**Pull request by IgorTodorovskiIBM**

### CODE REVIEW

File: docs/Guides/BashOnZOS.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+If Bash is not installed, you can install it using the following command:
+```
+zopen install bash
+```
+or you can download it directly from https://github.com/ZOSOpenTools/bashport/releases.
+
+3. Enter the following command to set Bash as your default shell in your OMVS segment:
+```
+tscmd "ALTUSER USERID omvs(program(/path/to/bash))
```
Comment (MikeFultonDev) [inline docs/Guides/BashOnZOS.md:35]: Can you point out that you may not have authority to issue this command and if not you should request your systems programmer perform it.

**Review by MikeFultonDev — COMMENTED** [review]
Just one comment. Looks great.

File: docs/Guides/BashOnZOS.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+```
+type bash
+```
+
+If Bash is not installed, you can install it using the following command:
+```
+zopen install bash
+```
+or you can download it directly from https://github.com/ZOSOpenTools/bashport/releases.
```
Comment (MikeFultonDev) [inline docs/Guides/BashOnZOS.md:31]: Do you want to add a link here to the section on how to set up zopen on your system ?

**Review by MikeFultonDev — COMMENTED** [review]
Added one more comment

File: docs/Guides/BashOnZOS.md (hunk 1 above)
Comment (IgorTodorovskiIBM) [reply docs/Guides/BashOnZOS.md]: Good point, added some details on that

**Review by MikeFultonDev — APPROVED** [review]

**Pull request merged**